### DISCUSSION TITLE
Can we open links in the browser? Or with the default app?

### DISCUSSION THREAD

**Discussion by rodhash** [MeanderingProgrammer/render-markdown.nvim#255] [General]
Can we open links in the browser? Or with the default app?

**Comment by MeanderingProgrammer** [comment]
Yes, but that's not something this plugin does, that's just part of neovim.

When your cursor is over the link destination, meaning anywhere between the parenthesis:

```
- [GitHub](https://github.com/MeanderingProgrammer/render-markdown.nvim)
           ^                                                          ^
```

Where the `^` symbols indicate the range your cursor can be in (inclusive), running the `gx` keymap in normal mode will open the link using your default app. More info on that is available in the help docs `:h gx`.

**Reply by rodhash** [reply]
Thanks.
